### DISCUSSION TITLE
change vue-cli (webpack) to vite

### DISCUSSION THREAD

**Discussion by potros** [directus/directus#5554] [Feature Requests]
I think replacing webpack with vite will help in making the development of the ui faster and a lot better

vite ;  https://vitejs.dev/

**Comment by tylerforesthauser** [comment]
If I'm not mistaken, the Vue3 branch has already made the switch 👍

**Reply by potros** [reply]
No

as you can see here : https://github.com/vuejs/vue-cli/blob/dev/package.json

vue-cli uses webpack not vite

Regards,

**Reply by tylerforesthauser** [reply]
Nah I meant this branch for _Directus_: https://github.com/directus/directus/tree/vue3

https://github.com/directus/directus/blob/vue3/app/package.json

**Reply by potros** [reply]
yes you're right

I really like what the team is doing in this project
